LabExT/Movement/PathPlanning.py

```diff
@@ -126,3 +129,158 @@ def _create_outline(
             y_max += self.FIBER_LENGTH
 
         return x_min, x_max, y_min, y_max
+
+
+class PotentialField:
```

Comment:
Generally: this class needs unittesting covering the algorithm math behind it. E.g. generate random waypoint trajectories and check for every time that both fibers are a minimum spacing apart.

Reply:
Will be solved in extra PR with simulation toolbox.